packages/components/src/components/navigation/LinkIcon/LinkIcon.stories.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { Fragment } from 'react';
+import { Meta } from '@storybook/react';
+import { LinkIcon } from './LinkIcon';
+import { linkIconSizes, linkIconVariants } from './types';
+import { Box, Table, Tbody, Td, Th, Thead, Tr } from '@chakra-ui/react';
+import { IconSend, Text } from '@components';
+
+export default {
+  component: LinkIcon,
+  title: 'Components/Navigation/LinkIcon',
+  argTypes: {
+    variant: {
+      options: [...linkIconVariants],
+      control: { type: 'select' },
+    },
+    size: {
+      options: linkIconSizes,
```

Comment:
Could we use `linkIconVariants` array directly? Like you did with `linkIconSizes`?